src/Shared/Components/DatePicker/types.ts

```diff
@@ -56,7 +56,7 @@ export interface MonthlySelectProps extends Pick<SingleDatePickerProps, 'dataTes
      *
      * @default 'new Date()'
      */
-    selectedMonthlyDate: OptionType
+    selectedMonthlyDate: SelectPickerOptionType
```

Comment:
We can create a base option type for DateTimePicker like:

SelectPickerOptionType<string>